portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Message.path

```diff
@@ -270,6 +270,11 @@
 	<td>//div[@role='alert' and contains(.,'Please select a valid date range.')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>ERROR_MULTIPLE_UPLOAD_VALID</td>
```

Comment:
valid or invalid?

Reply:
it's not clear what this path is trying to assert. can you change the name @Hanlf ?